server/routerlicious/packages/lambdas/src/alfred/index.ts

```diff
@@ -301,6 +302,25 @@ export function configureWebSocketServices(
 				throw new NetworkError(403, "Must provide an authorization token");
 			}
 
+			// Check if current cluster is in draining
+			if (clusterDrainingChecker) {
+				let clusterInDraining = false;
```

Comment:
We should perform this check after the auth is done.

Reply:
Normally yes. Here isClusterDraining can be a in memory call (like checking the flag in memory) and it is cheaper than the token validation flow. And we would prefer the cheaper call happens first? Please share your thoughts. Thanks

Reply:
It's more of a security concern. Without an auth, we are leaking info to unauthorized callers, and we don't know how they could exploit this info. Besides, in the future, we may change the implementation to make it more expensive, and it's hard for the developer to make the change here.